src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
 // write your code here
+
+const table = document.querySelector('table');
+
+for (let i = 0; i < table.rows.length; i++) {
+  const row = table.rows[i];
+  const position = row.cells[1];
+  const salary = row.cells[4];
```

Comment:
```suggestion
  const salary = row.cells[4];
```
В завданні сказано
> append it in front of the last table column

А що, якщо кількість колонок нам би не була відома? Тоді б `cells[4]` не підійшло б. Давай зробимо ріщення більш універсальним і використаємо `lastElementChild.before`

Reply:
Дякую! [Виправив](https://github.com/mate-academy/js_task_clone_table_column_DOM/pull/211/commits/2cf7e31495bea3f0eb2761845c2c743294a2cc47)

Reply:
Не помітив спершу "before". Зараз ще дороблю.

Reply:
[Доробив з before](https://github.com/mate-academy/js_task_clone_table_column_DOM/pull/211/commits/9c000ae91bface61293a33d47dc0948c2c74935f). Тепер вже остаточно :)